addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -27,6 +28,16 @@
     &:not(.o-composer-is-compact) {
         // Don't allow the input to take the whole height when it's not compact
         // (like in chatter for example) but allow it to take some more place
-        max-height: 150px;
+        max-height: 400px;
     }
 }
+
+.o_ComposerTextInput_textarea_dummy {
+    position: absolute;
+    opacity: 0;
+    border-top-width: 0;
+    border-bottom-width: 0;
+    overflow: hidden;
+    top: -10000;
+    direction: -10000;
```

Comment:
You probably copied it, but that doesn't look like a valid value for direction, I have no idea why we would need that